pkg/kwokctl/cmd/delete/cluster/cluster.go

```diff
@@ -52,19 +54,24 @@ func NewCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", flags.Kubeconfig, "The path to the kubeconfig file that will remove the deleted cluster")
+	cmd.Flags().BoolVar(&flags.Force, "force", flags.Force, "Force delete the cluster managed by kwokctl")
 	return cmd
 }
 
 func runE(ctx context.Context, flags *flagpole) error {
-	name := config.ClusterName(flags.Name)
-	workdir := path.Join(config.ClustersDir, flags.Name)
+	return deleteCluster(ctx, flags.Name, flags.Kubeconfig, flags.Force)
+}
+
+func deleteCluster(ctx context.Context, clusterName string, kubeconfigPath string, force bool) error {
+	name := config.ClusterName(clusterName)
+	workdir := path.Join(config.ClustersDir, clusterName)
```

Comment:
There seems to be no need to change

Reply:
I just added the new function for some better code modularity, can get it back to normal if you want.